src/Pokemon.java

```diff
@@ -0,0 +1,33 @@
+public abstract class Pokemon {
+    private String name;
+    private int level;
+
+    public Pokemon(String name, int level) {
+        this.name = name;
+        this.level = level;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public int getLevel() {
+        return level;
+    }
+
+    public void setLevel(int level) {
+        this.level = level;
+    }
+
+    public void defend() {
+        System.out.println(this.name + " is defending!");
+    }
+
+    // This is the abstract method that all subclasses must implement
```

Comment:
Dit commentaar is handig voor jezelf, maar let er op dat dit straks niet in je eindopdracht thuis hoort. Ik kan zien dat dit een abstracte methode is, dus het commentaar is overbodig. In de regel wil je commentaar zo min mogelijk. Je wilt dat je code leesbaar genoeg is, zodat commentaar niet nodig is.